_schemaV2.graphql

```diff
@@ -7263,6 +7268,23 @@ type Me implements Node {
   unreadNotificationsCount: Int!
 }
 
+# Collection of fields that describe medium type, such as _Painting_. (This field
+# is also commonly referred to as just "medium", but should not be confused with
+# the artwork attribute called `medium`.)
+type MediumType {
+  # A globally unique ID.
+  id: ID!
```

Comment:
odd that this type has id fields?

Reply:
`artworkClassifications` have IDs as well. Wasn't sure if relay would make a fuss without them.

Reply:
interesting! in that case i would probably keep the `id: ID!`(but drop the `internalID` cause thats probably not needed).